### DISCUSSION TITLE
Opendatacam Multi Camera Issue

### DISCUSSION THREAD

**Discussion by Enes-Dogukan-NTT** [opendatacam/opendatacam#530] [Q&A] [answered]
I am trying to use 3 indipendent camera with opendatacam YoloV4 Tiny model on Jetson Nano. But I am considering GPU problem of Jetson Nano. Is it possible to use opendatacam YoloV4 Tiny model with multiple indipendent camera on Nvidia Jetson Nano ? Therefore, if jetson nano is not support 3 indipendent camera is Xavier NX enough to solve this problem ?

**Comment by SerdarHelli** [comment]
Yeah I am considering too

**Comment by vsaw** [comment, marked as answer]
Jetson Nano runs a single Yolv4-tiny at approx ~15 fps. So running multiple independent instances will not be possible.

The Xavier NX can run Yolvo4-tiny at ~60fps, so there things could work out and you may be able to run 3 instances at ~20fps. However, I have never tried it and don't know if you will run into other limitations (e.g. RAM).
